apps/examples/testcase/le_tc/network/tc_net_close.c

```diff
@@ -77,8 +115,10 @@ static void tc_net_close_n(void)
 
 int net_close_main(void)
 {
-
 	tc_net_close_p();
+	tc_net_close_multi_times_p();
```

Comment:
This is not an utc. This is an itc. You should name it as itc_xxx. And please put itcs at separated location from utc.
```
tc_xxx1
tc_xxx2

itc_yyy1
itc_yyy2
```